core/plugins/org.polarsys.capella.core.data.migration/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.2"?>
 <!--
-  Copyright (c) 2006, 2021 THALES GLOBAL SERVICES.
+  Copyright (c) 2006, 2022 THALES GLOBAL SERVICES.
```

Comment:
Copyright must be changed before merge.
Copyright (c) 2006, 2022 THALES GLOBAL SERVICES and others.